system/hardware/hardwared.py

```diff
@@ -34,6 +34,7 @@
 TEMP_TAU = 5.   # 5s time constant
 DISCONNECT_TIMEOUT = 5.  # wait 5 seconds before going offroad after disconnect so you get an alert
 PANDA_STATES_TIMEOUT = round(1000 / SERVICE_LIST['pandaStates'].frequency * 1.5)  # 1.5x the expected pandaState frequency
+ONROAD_CYCLE_TIME = 1  # seconds to wait offroad after requesting an onroad cycle
```

Comment:
why is one cycle not enough?

Reply:
partially because 1 second looked better (didn't immediately exit toggles to onroad so you could review your change/toggle another), and if manager lags it won't see the one frame with started=False since it always conflates

Reply:
separately, 1 frame more obviously exposes our slow manager shutdown. ui shows waiting for controls because it measures start time while manager is still shutting down processes